gabrielgomesml/training_project_client
# Feature/home

## What I did

- Create Homepage;
- Debounce hook;
- Navbar;
- Fix auth problems;
- Create Admin page;

## How to test

- First of all make sure you are running the most updated server version in your machine. You can get it here: https://github.com/gabrielgomesml/training_project_server/pull/2
- Open the project and use the command `nvm use`
- Run a `yarn` or `yarn install`
- After that run the command `yarn start` to start the project
- Make sure you have users (with different role property: 0 or 1) and movies related to them. I recommend you test this PR first https://github.com/gabrielgomesml/training_project_server/pull/2
- Log into an user account with movies and test the search bar with movies titles and click in one of them. Should appear a modal with the movie infos. Also a carousel is on it but not connected with database yet;
- If you are in an admin account (role 1) you have in the navbar a section called "Admin" (this section should not appear for comum users and if they tried to access the admin page with the url (localhost:8080/admin), should be redirect to the homepage;
- This page has all users listed and you can block they access to the system;
- After all, click in your picture of navbar and sign out of the system.
